### PR TITLE
Feature/stripe

### PR DESCRIPTION
**Pull request by r0odr1**
## Pull Request: Adding Payment Functionality via Stripe

**Description:**

Payment method implementation.

**Testing**

- The payment method test was conducted, and it was successful.

**Screenshots:**

image

### CODE REVIEW

File: src/api/checkout.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+
+const URL = `${import.meta.env.VITE_BASE_URL}`;
+
+export const postCheckout = async (form) => {
+  try {
+    console.log(form);
```
Comment (ht1204) [inline src/api/checkout.js]: please, remove this console.log

File: src/pages/Summary/index.jsx [hunk 2]
```diff
@@ -29,45 +37,55 @@ const Checkout = () => {
     dispatch(clearCart());
     navigate(-1);
   };
-
+  console.log(payment);
```
Comment (ht1204) [inline src/pages/Summary/index.jsx]: remove this console.log

File: src/pages/Summary/index.jsx [hunk 3]
```diff
@@ -1,17 +1,25 @@
...
+import AlwaysFirst from "../../components/AlwaysFirst";
+import Cart from "../../components/Cart";
+import { Elements } from "@stripe/react-stripe-js";
+import { loadStripe } from "@stripe/stripe-js";
+import { selectPayment } from "../../store/redux/slices/cartSlice";
+import { useSelector, useDispatch } from "react-redux";
+import "./index.scss";
+
+const stripePromise = loadStripe(
+  "pk_test_51NiN00Isblv2sRW8edxjWByXYrbDZG86nyVa4KDjJCMLZi5QaOjiqlOb02jafd0jRDWI3OtrQmr3XCS94ILcFDLJ00y4dMEfMb"
```
Comment (ht1204) [inline src/pages/Summary/index.jsx]: this key should be better in a constant for importing.

**Comment by r0odr1** [comment]
resolved

**Review by ht1204 — APPROVED** [review]

**Pull request merged**